notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -356,3 +356,46 @@ demon [Bill and Ted met a pair of rather friendly demons during their brief stay
 reconciliation [Bill and Ted patched things up with the Grim Reaper who'd had a falling out with the pair due to a lawsuit.]
 
 
+movie: 2067 (2020)
+==================
+
+:: Title
+2067
+
+:: Description
+In the year 2067, Earth has been devastated by climate change, and it is up to
+one man to travel 407 years into the future to set things straight.
+
+:: Date
+2020-10-02
+
+:: References
+https://en.wikipedia.org/wiki/2067_(film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+what if the fate of the world was in my hands [It was up to Ethan to travel 407 years into the future to find a cure for a strange sickness that was going to wipe out humanity.]
+global warming [The film is set 2067 when Earth has been devastated by climate change. All plant life on Earth had gone extinct, and civilization had collapsed around the globe with only one city in Australia still holding out.]
+time machine [A main novelty of the film is the large, nuclear powered, stationary time machine that Ethan used to travel 407 years into the future.]
+husband and wife [Ethan was caring for his ill wife, Xanthe, and missed her deeply after he traveled into the future.]
+coping with a loved one being gravely ill [Ethan's beloved wife, Xanthe, was dying from a strange affliction that was a result of her breathing tainted oxygen.]
+destiny [Ethan was convinced that it was his fate (a word that is uttered five times in the film) that Jude would shoot him dead, and in so doing put an end to Ethan's effort to save humanity. Toward the end, however, Ethan changed his tune an resolved to change his own fate, and the fate of humanity by extension.]
+father and son [Central to the story was Ethan's dealing with his daddy issues.]
+
+:: Minor Themes
+coping with being in a heat wave [A reporter spoke of a record-breaking heat wave.]
+hologram [People communicated via holographic recordings.]
+suicide [A guilt-ridden Jude took his own life. It was said that Ethan's father had committed suicide when Ethan was only eight years old. Although it was later revealed that the Chronicorp CTO had shot him dead.]
+love vs. the public good [Should Ethan stay in the present to care for his gravely ill wife or go on a possibly one way trip into the future in a last ditched effort to save humanity?]
+wilderness survival [Ethan rubbed two sticks together to make a fire to survive the night in the rainforest.]
+personal identity [Ethan was puzzled about if his future skeleton was him.] {48:30}
```

Comment:
I guess this {} time mark should not be merged into master?

Reply:
Whoops. That's one of my time cues to remind me to go back and figure out what was going on later. Neglected to erase that one before my initial commit. It's gone now.